src/views/Stake/Stake.tsx

```diff
@@ -418,96 +419,57 @@ function Stake() {
+                      {...{ isAppLoading }}
+                    />
+                    <StakeRow
+                      title="Single Staking"
+                      balance={`${trim(Number(sohmBalance), 4)} sOHM`}
+                      indented
+                      {...{ isAppLoading }}
+                    />
+                    <StakeRow
+                      title="Staked Balance in Fuse"
+                      balance={`${trim(Number(fsohmBalance), 4)} fsOHM`}
+                      indented
+                      {...{ isAppLoading }}
+                    />
+                    <StakeRow
+                      title="Wrapped Balance"
+                      balance={`${trim(Number(wsohmBalance), 4)} wsOHM`}
+                      {...{ isAppLoading }}
+                      indented
+                    />
+                    <StakeRow
+                      title="Wrapped Balance in FiatDAO"
+                      balance={`${trim(Number(fiatDaowsohmBalance), 4)} wsOHM`}
+                      {...{ isAppLoading }}
+                      indented
+                    />
+                    <StakeRow
+                      title="Wrapped Balance (v2)"
+                      balance={`${trim(Number(gOhmBalance), 4)} gOHM`}
+                      indented
+                      {...{ isAppLoading }}
+                    />
                     <Divider color="secondary" />
-
-                    <div className="data-row">
-                      <Typography variant="body1">
-                        <Trans>Next Reward Amount</Trans>
-                      </Typography>
-                      <Typography variant="body1">
-                        {isAppLoading ? <Skeleton width="80px" /> : <>{nextRewardValue} sOHM</>}
-                      </Typography>
-                    </div>
-
-                    <div className="data-row">
-                      <Typography variant="body1">
-                        <Trans>Next Reward Yield</Trans>
-                      </Typography>
-                      <Typography variant="body1">
-                        {isAppLoading ? <Skeleton width="80px" /> : <>{stakingRebasePercentage}%</>}
-                      </Typography>
-                    </div>
-
-                    <div className="data-row">
-                      <Typography variant="body1">
-                        <Trans>ROI (5-Day Rate)</Trans>
-                      </Typography>
-                      <Typography variant="body1">
-                        {isAppLoading ? <Skeleton width="80px" /> : <>{trim(Number(fiveDayRate) * 100, 4)}%</>}
-                      </Typography>
-                    </div>
+                    <StakeRow title="Next Reward Amount" balance={`${nextRewardValue} sOHM`} {...{ isAppLoading }} />
+                    <StakeRow title="Next Reward Yield" balance={`${stakingRebasePercentage}%`} {...{ isAppLoading }} />
+                    <StakeRow
+                      title="ROI (5-Day Rate)"
```

Comment:
as an example for my other comment this could be 

```javascript
title={t`ROI (5-Day Rate)`}
```

Some example usage here: https://github.com/OlympusDAO/olympus-frontend/blob/develop/src/components/TopBar/LocaleSwitch.tsx#L37

https://lingui.js.org/ref/macro.html#t